resources/zib/zib-Procedure.xml

```diff
@@ -0,0 +1,209 @@
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Procedure" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Procedure" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Procedure">
+      <path value="Procedure" />
+      <short value="Procedure" />
+      <definition value="The zib Procedure is mapped for all but one concept (Requester) to this Procedure profile and a profile on ServiceRequest (http://nictiz.nl/fhir/StructureDefinition/zib-Procedure-ServiceRequest). The Requester is mapped in the ServiceRequest profile.&#xD;&#xA;&#xD;&#xA;Practically, the zib Procedure is split in two based on time to align with the intention of FHIR. All past procedures are covered in the Procedure resource. All future procedures, including the adviced procedures, are covered in the ServiceRequest resource. Both resources contain the zib mappings. The only exeption is the Requester concept which is only mapped in ServiceRequest." />
+      <alias value="Verrichting" />
+      <mapping>
+        <identity value="zib-procedure-v5.2-2020EN" />
+        <map value="NL-CM:14.1.1" />
+        <comment value="Procedure" />
+      </mapping>
+    </element>
+    <element id="Procedure.extension">
+      <path value="Procedure.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Procedure.extension:procedureMethod">
+      <path value="Procedure.extension" />
+      <sliceName value="procedureMethod" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/procedure-method" />
+      </type>
+    </element>
+    <element id="Procedure.extension:procedureMethod.value[x]">
+      <path value="Procedure.extension.value[x]" />
+      <short value="ProcedureMethod" />
+      <definition value="the method or technique that was used to perform the procedure, e.g. approach, lavage, pressuring, ets" />
+      <alias value="VerrichtingMethode" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.14.1.4--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-procedure-v5.2-2020EN" />
+        <map value="NL-CM:14.1.12" />
+        <comment value="ProcedureMethod" />
+      </mapping>
+    </element>
+    <element id="Procedure.basedOn">
+      <path value="Procedure.basedOn" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/CarePlan" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/ServiceRequest" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Procedure-ServiceRequest" />
+      </type>
+    </element>

```

Comment:
Maybe add some guidance on how/why this is added?

Reply:
I placed that on the root... do we need to copy or move it?

Reply:
Missed that. Copying is not necessary
Definition on the root does not mention how the two can relate to each other, which is implicit by this targetProfile. Do we need to make it explicit? Not sure actually.

Reply:
Just made it more explicit! Can you have another look if this is sufficient?

Reply:
I think it would be valuable to point to root comment here.